### DISCUSSION TITLE
Picture alt text

### DISCUSSION THREAD

**Discussion by tboulis** [ShapeCrawler/ShapeCrawler#640] [Q&A] [answered]
Hello, is it possible to read, add or update the alt text of an image? The corresponding property for the xml is `descr` - here's the link to the documentation for it: https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.presentation.nonvisualdrawingproperties.description?view=openxml-3.0.1#documentformat-openxml-presentation-nonvisualdrawingproperties-description

**Comment by AdamShakhabov** [comment, marked as answer]
Hello. This is not available yet. I've created an issue for this implementation: https://github.com/ShapeCrawler/ShapeCrawler/issues/644

**Reply by tboulis** [reply]
Thank you @ashahabov <3
